.platform/routes.yaml

```
@@ -276,10 +276,89 @@ https://docs.upsun.com/:
+      "/anchors/configuration/yaml.html": { "to": "/learn/overview/yaml.html", "code": 301, "prefix": false }
+      "/anchors/apps/mounts.html": { "to": "/create-apps/app-reference.html#mounts", "code": 301, "prefix": false }
+      "/anchors/apps.html": { "to": "/create-apps/app-reference.html", "code": 301, "prefix": false }
+      "/anchors/backups.html": { "to": "/security/backups.html", "code": 301, "prefix": false }
+      "/anchors/admin/users.html": { "to": "/administration/users.html", "code": 301, "prefix": false }
+      # No equivalent yet for Upsun.
+      "/anchors/wcag.html": { "to": "https://platform.sh/wcag/", "code": 301, "prefix": false }
+      "/anchors/observability/metrics.html": { "to": "/increase-observability/metrics.html", "code": 301, "prefix": false }
+      # Repeated again for the following
+      # "links.documentation.metrics": "https://docs.platform.sh/increase-observability/metrics.html",
+      "/anchors/observability/metrics/grid.html": { "to": "/increase-observability/metrics/grid.html", "code": 301, "prefix": false }
+      # Not relevant for Upsun.
+      "/anchors/observability/metrics/dedicated.html": { "to": "https://docs.platform.sh/increase-observability/metrics/dedicated.html", "code": 301, "prefix": false }
+      "/anchors/observability/metrics/dedicated-gen-3.html": { "to": "https://docs.platform.sh/increase-observability/metrics/dedicated-generation-3.html", "code": 301, "prefix": false }
+      "/anchors/integrations/activities.html": { "to": "/integrations/activity.html", "code": 301, "prefix": false }
+      "/anchors/integrations/activity-scripts.html": { "to": "/integrations/activity.html#configuring-scripts", "code": 301, "prefix": false }
+      "/anchors/integrations/source.html": { "to": "/integrations/source.html", "code": 301, "prefix": false }
+      "/anchors/apps/timezone.html": { "to": "/configuration/app/timezone.html", "code": 301, "prefix": false }
+      "/anchors/organizations.html": { "to": "/administration/organizations.html", "code": 301, "prefix": false }
+      "/anchors/redeploy.html": { "to": "/development/troubleshoot.html#force-a-redeploy", "code": 301, "prefix": false }
+      "/anchors/ssh/authenticate.html": { "to": "/development/ssh.html#authenticate-with-ssh-keys", "code": 301, "prefix": false }
+      "/anchors/apps/workers.html": { "to": "/create-apps/workers.html", "code": 301, "prefix": false }
+      "/anchors/apps/source-ops.html": { "to": "/create-apps/source-operations.html", "code": 301, "prefix": false }
+      "/anchors/apps/runtime-ops.html": { "to": "/create-apps/runtime-operations.html", "code": 301, "prefix": false }
+      "/anchors/live-backups.html": { "to": "/administration/backup-and-restore.html#live-backups", "code": 301, "prefix": false }
+      "/anchors/automated-backups.html": { "to": "/learn/tutorials/dependency-updates.html", "code": 301, "prefix": false }
```

Shouldn't this one be:

```suggestion
      "/anchors/automated-updates.html": { "to": "/learn/tutorials/dependency-updates.html", "code": 301, "prefix": false }
```
?